ts/Data/DataConverter.ts

```diff
@@ -197,6 +547,22 @@ namespace DataConverter {
     export type Type = (
         boolean|null|number|string|Date|DataTable|undefined
     );
+
+    /**
+     * The shared options for all DataParser instances
+     */
+    export interface Options extends DataJSON.JSONObject {
+        dateFormat?: string;
+        alternativeFormat?: string;
+    }
```

Comment:
If these refactored options are used only by parsers, it should be part of the base class there and not be part of the data core.

Reply:
Options are used by DataConverter. The comment is wrong, should be `Internal options for DataConverter`. I will fix it.